app/config/collections.php

```diff
@@ -1894,14 +1894,14 @@
                 'attributes' => ['search'],
                 'lengths' => [2048],
                 'orders' => [Database::ORDER_ASC],
-            ],
+            ]
         ],
     ],
 
-    'tags' => [
+    'deployments' => [
```

Comment:
Make sure to update usage worker that calculates the storage taken by tags

Reply:
+1